src/inc/DefaultSettings.h

```diff
@@ -30,7 +30,7 @@ constexpr short DEFAULT_HISTORY_SIZE = 9001;
 #pragma warning(push)
 #pragma warning(disable : 26426)
 // TODO GH 2674, don't disable this warning, move to std::wstring_view or something like that.
-const std::wstring DEFAULT_FONT_FACE{ L"Consolas" };
+const std::wstring DEFAULT_FONT_FACE{ L"Cascadia Code" };

```

Comment:
Oh danger here - this might break the WpfTerminal that VS is using, unless they're also shipping Cascadia Code or Cascadia Code is present in the package - we might have to leave this one